### PR TITLE
[risksense-835] Contrib/crestdatasys risksense minor fix

### PR DESCRIPTION
**Pull request by crestsantoshgiri**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: Changes in Test Playbook.

## Description
Minor fixes: changed exclusivity to exclude in playbook*-Test.yml(s)

## Minimum version of Demisto
- [ ] 4.5.0
- [x] 5.0.0
- [ ] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation 

## Demisto Partner?
- [x] The title must be in the following format: **[RiskSense-835] Minor fixes related to Test playbooks**

### CODE REVIEW

File: Packs/RiskSense/ReleaseNotes/1_0_2.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Test Playbooks
+##### Change in argument
+ - Argument exclusivity is changed to exclude.
```
Comment (anara123) [inline Packs/RiskSense/ReleaseNotes/1_0_2.md:4]:
```suggestion
#### Integrations
##### __RiskSense__
Errors messages updated
```

**Comment by anara123** [comment]
@kirbles19 please review

**Comment by crestdatasystems** [comment]
@anara123 we're good on this, no change is required.

**Comment by kirbles19** [comment]
@anara123 Done.

**Pull request merged**